### DISCUSSION TITLE
DPX YCbCr 4:2:2

### DISCUSSION THREAD

**Discussion by protean-ctl** [ImageMagick/ImageMagick#2817] [Digital Image Processing]
Hi,

I was trying to convert 10bit rgb DPXs to YCbCr 4:2:2 but all my tests come out with a 4:4:4 configuration. GraphicsMagick gave me something but it's CbYCr. I have to say I'm not very familiar with these chroma subsampled images and only recently discovered DPX could be 4:2:2.

Is there a way to use IM to get 4:2:2 DPXs and is there an expected component configuration for such a file?

Here's an example.
magick convert -colorspace rec709YCbCr -sampling-factor 4:2:2 <inputfile> <outputfile>.

Any help is greatly appreciated!

j

**Comment by urban-warrior** [comment]
Your command is correct but it looks like you discovered a bug.  We'll need a few days to identify the problem and issue a patch.  Thanks for alerting us to the problem.
